internal/pkg/agent/migration/migrate_config_test.go

```diff
@@ -0,0 +1,273 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build linux || windows
+// +build linux windows
+
+package migration
+
+import (
+	"bytes"
+	"io"
+	"io/fs"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/elastic-agent-libs/logp"
+	"github.com/elastic/elastic-agent/internal/pkg/agent/application/paths"
+	"github.com/elastic/elastic-agent/internal/pkg/agent/application/secret"
+	"github.com/elastic/elastic-agent/internal/pkg/agent/storage"
+)
+
+type configfile struct {
+	name        string
+	create      bool
+	permissions fs.FileMode
+	content     []byte
+}
+
+func TestMigrateToEncryptedConfig(t *testing.T) {
+
```

Comment:
please remove newlines after `func` and `t.Run`s, to be consistent with code everywhere